cleverpush/src/main/java/com/cleverpush/stories/models/Story.java

```diff
@@ -0,0 +1,64 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class Story implements Serializable {
+
+    @SerializedName("_id")
+    @Expose
+    private String id;
+    @SerializedName("channel")
+    @Expose
+    private String channel;
+    @SerializedName("title")
+    @Expose
+    private String title;
+    @SerializedName("content")
+    @Expose
+    private Content content;
+    private boolean isOpened = false;
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getChannel() {
+        return channel;
+    }
+
+    public void setChannel(String channel) {
+        this.channel = channel;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public Content getContent() {
+        return content;
+    }
+
+    public void setContent(Content content) {
+        this.content = content;
+    }
+
+    public boolean isOpened() {
+        return isOpened;
+    }
+
+    public void setOpened(boolean opened) {
+        isOpened = opened;
```

Comment:
please use this, as you want to reference to the class scope variable (make it more descriptive)

also, see, you even used opened in the parameter name. ;)